test/clanhr/reply/core_test.clj

```diff
@@ -58,3 +60,6 @@
     (is (= 401 (:status response-unauthorised)))
     (is (= 400 (:status response-bad)))))
 
+(deftest async-reply-test
+  (let [response (reply/ok {})]
+    (is (d/deferred? (reply/async-reply response)))))
```

Comment:
I think that these functionality is provided by aleph instead of manifold.deferred.